docs/guides/best_practices/performance.md

```diff
@@ -19,6 +20,11 @@ it has not seen, it will compute the predictions and store them in a cache. The
 next time it is called with the same parameters, instead of recomputing the
 predictions, it will return the previously computed value from the cache.
 
+Alternatively, you may use Python's builtin `functools` library to cache
+expensive computations. It is important to note that `mo.save.cache` is more
+expensive than `functools.cache`, and there are certain instances where
+`functools.cache` is more appropriate.
```

Comment:
It's not clear to me as a user how to decide between using `mo.save.cache` and using `functools.cache`.